### PR TITLE
Add icons for Mark as read and unread

### LINKED ISSUE #5892: Add icons for Mark as read and unread
**Is your feature request related to a problem? Please describe.**
There are no icons for both the "Mark as read" and "Mark as unread" context menu options which looks odd next to the other options which do have icons.

**Describe the solution you'd like**
Icons should be added for the "Mark as read" and "Mark as unread" options.

**Additional context**

image

image

### PR DESCRIPTION
**Pull request by Pytal**
Fix #5892

image

image

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: src/components/LeftSidebar/ConversationsList/Conversation.vue [hunk 1]
```diff
@@ -59,6 +59,7 @@
 				{{ t('spreed', 'Copy link') }}
 			</ActionButton>
 			<ActionButton
+				icon="icon-checkmark"
```
Comment (PVince81) [inline src/components/LeftSidebar/ConversationsList/Conversation.vue:62]: use a material design icon here as well ?

File: src/components/MessagesList/MessagesGroup/Message/Message.vue [hunk 2]
```diff
@@ -149,6 +149,9 @@ the main body of the message as well as a quote.
 							<ActionButton
 								:close-after-click="true"
 								@click.stop="handleMarkAsUnread">
+								<template #icon>
+									<CheckCircleOutline size="20" />
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/Message/Message.vue:153]:
```suggestion
									<CheckCircleOutline
										decorative
										title=""
										:size="16" />
```

Not a too big fan of the icon, but I guess it clarifies the action and better than no icon

**Review by marcoambrosini — CHANGES_REQUESTED** [review]
I don't the unread icon is appropriate, what about `eye-off-outline`?

eye-off-outline

**Comment by nickvergessen** [comment]
Moved both to material icons and using outline eye now:

Mark read | Mark unread
---|---
Bildschirmfoto von 2021-06-30 09-07-38 | Bildschirmfoto von 2021-06-30 09-07-34

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**